modules/drivers/athena/test/metabase/test/data/athena.clj

```diff
@@ -28,8 +29,8 @@
 ;; Athena doesn't support dashes in Table names... we'll just go ahead and convert them all to underscores, even for DB
 ;; names.
 (defmethod ddl.i/format-name :athena
-  [driver table-or-field-name]
-  ((get-method ddl.i/format-name :sql-jdbc) driver (str/replace table-or-field-name #"-" "_")))
+  [driver database-or-table-or-field-name]
+  ((get-method ddl.i/format-name :sql-jdbc) driver (str/replace database-or-table-or-field-name #"-" "_")))
```

Comment:
Unrelated clarification, but this is used for database names too, at least in this namespace